### PR TITLE
Social distancing eval coral dev

### PR DESCRIPTION
**Pull request by mrn-mln**
#102 To solve this issue I implemented another module to evaluate edgetpu models. 
Readme will be updated.

### CODE REVIEW

File: applications/smart-distancing/eval/Dockerfile-amd64-usbtpu-eval [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# docker can be installed on the dev board following these instructions:
+# 1) build: docker build -f Dockerfile-amd64-usbtpu -t "neuralet/amd64:eval-usbtpu" .
```
Comment (undefined-references) [inline applications/smart-distancing/eval/Dockerfile-amd64-usbtpu-eval:2]: docker build -f Dockerfile-amd64-usbtpu-eval -t "neuralet/amd64:eval-usbtpu" .

File: applications/smart-distancing/eval/Dockerfile-amd64-usbtpu-eval [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+RUN apt-get install -y python3-opencv python3-scipy
+
+WORKDIR /repo/applications/smart-distancing/eval
+
+# --model_path: tflite model path | --classes: the classes text file path | --minscore: the minimum confidence for detecting objects
+# --img_path: validation set directory | --input_size: the input size of the model | --result_dir: the detection results with export here
+# -gt: the groundtruth path | -t: minimum IOU threshold
+
+ENTRYPOINT ["python3", "edgetpu_eval.py"]
+CMD ["--model_path", "edgetpu/data/mobilenet_v1_face_mask_edgetpu.tflite", "--classes", "eval_files/sample_classes.txt","--minscore", "0.25", "--img_path", "val_images", "--input_size", "300,300,3", "--result_dir", "eval_files/detresults/", "-gt", "eval_files/groundtruths","-t", "0.5" ]
```
Comment (undefined-references) [inline applications/smart-distancing/eval/Dockerfile-amd64-usbtpu-eval:30]: ValueError: Could not open 'edgetpu/data/mobilenet_v1_face_mask_edgetpu.tflite'

I couldn't find any download script in root of this directory or download url in Readme for `mobilenet_v1_face_mask_edgetpu.tflite` 
It is needed if user just want to test the docker and scripts

File: applications/smart-distancing/eval/Dockerfile-eval [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# docker can be installed by following instructions: 
+# build: docker build -f Dockerfile-amd64usbtpu-eval -t edgetpu-detector-eval .
```
Comment (undefined-references) [inline applications/smart-distancing/eval/Dockerfile-eval]: There is name conflicts in build command and dockerfile name.

File: applications/smart-distancing/eval/Dockerfile-amd64-usbtpu-eval (hunk 2 above)
Comment (mrn-mln) [reply applications/smart-distancing/eval/Dockerfile-amd64-usbtpu-eval:30]: @emma-w-dev evaluator module is a tool and users who want to use it should have their own tflite models (for edgetpu devices) and ground truths for evaluating.

File: applications/smart-distancing/eval/README.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+# Smart Social Distancing - Performance Metrics for Object Detection
```
Comment (undefined-references) [inline applications/smart-distancing/eval/README.md:1]: I found that there are two possible workflow in this repo:

1. To Evaluate the detection result which are ready to use (in the format you have explained in Readme)
2. To run tflite model using `edgetpu_eval.py` and test data to generate test result and then evaluate results

but I can't find these information in the Readme file. please provide more basic information in Readme about the workflow of provided scripts and dockers.

Comment (mrn-mln) [reply applications/smart-distancing/eval/README.md:1]: Yes, other workflows will be provided such as test on coral-dev devices, jetson-nano, GPU, ...

**Comment by mhejrati** [comment]
@mrn-mln what is the status of this? Shall we close it for now?

**Comment by mrn-mln** [comment]
@mhejrati I closed it because of [a new feature ](https://github.com/neuralet/smart-social-distancing/pull/19)for evaluating detector results is implemented.

**Pull request closed without merging**